test/test_helper.rb

```diff
@@ -11,12 +11,17 @@
 require 'date'
 require 'holidays'
 require 'holidays/core_extensions/date'
+require 'holidays/core_extensions/time'
 
 # Loads core extension for use in various definition tests as necessary
 class Date
   include Holidays::CoreExtensions::Date
 end
 
+class Time
```

Comment:
This is a good addition, can you also add some specific tests around the new `Date` and `Time` core extensions? Right now those tests aren't great so feel free to create a new test file just to test the new fields. I would then probably refactor to migrate the core ext tests to your new file (if we go in that direction).